.github/workflows/deploy.yml

```diff
@@ -0,0 +1,32 @@
+name: Deployment
+
+on:
+  release:
+    types: [created]
+    branches: [ master ]
+
+jobs:
+  deploy:
+
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v2
+      - name: Set up JDK 8
+        uses: actions/setup-java@v2
+        with:
+          java-version: '8'
+          distribution: 'adopt'
```

Comment:
The other file says "cache: maven", any reason for doing that there and not here?

Reply:
No, not really. I'll add it.